src/features/features.controller.ts

```diff
@@ -196,7 +207,7 @@ export class FeaturesController {
         } else {
             throw new HttpException(
                 "user is unauthorized to perform this action",
```

Comment:
```suggestion
                "Access denied: You do not have sufficient permissions to perform this action",
```

Reply:
Currently in some controllers there are no permissions for own_team , after the permissions are added then, we can add the forbidden tests to it.